### PR TITLE
ANA-1915: Use identical effectiveAt within GetComplexMarketData requests

### PR DESCRIPTION
**Pull request by peturparkur-fbn**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Fix integration tests that would break with the addition of ANA-1753 which forces get_complex_market_data endpoint effectiveAt to be identical to the complex_market_data_id.effectiveAt within the request

### CODE REVIEW

File: examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb [hunk 1]
```diff
@@ -1393,7 +1395,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3.10.6 64-bit",
```
Comment (SamuelColvinFinbourne) [inline examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1398]: Are these changes intentional? it looks like it has replaced some things with information that is local to your machine?

File: examples/use-cases/instruments/Forward Rate Agreement.ipynb [hunk 2]
```diff
@@ -45,8 +45,9 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "/Users/peternagymathe/secrets/fbn-ci+local-sdk-tests.json\n",
```
Comment (SamuelColvinFinbourne) [inline examples/use-cases/instruments/Forward Rate Agreement.ipynb:48]: Yeah, this is definitely something you don't want to commit

File: examples/use-cases/instruments/Forward Rate Agreement.ipynb [hunk 3]
```diff
@@ -1229,7 +1232,8 @@
        "      <th>total_consideration.currency</th>\n",
        "      <th>exchange_rate</th>\n",
        "      <th>transaction_currency</th>\n",
-       "      <th>properties</th>\n",
+       "      <th>properties.Transaction/default/ParentLuid.key</th>\n",
+       "      <th>properties.Transaction/default/ParentLuid.value.label_value</th>\n",
```
Comment (SamuelColvinFinbourne) [inline examples/use-cases/instruments/Forward Rate Agreement.ipynb:1236]: This surprises me too, I didn't expect to see this change

File: examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb (hunk 1 above)
Comment (peturparkur-fbn) [reply examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1398]: ah yes, whoops. I'll try to figure out how to remove them

File: examples/use-cases/instruments/FX Forward.ipynb [hunk 4]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (rizwansaeed) [inline examples/use-cases/instruments/FX Forward.ipynb:2]: <p>can you remove the <code>/Users/peternagymathe/secrets/fbn-ci+local-sdk-tests.json</code> output </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/573/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/FX Forward.ipynb:2]: <p>could you remove this output too </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/573/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/FX Forward.ipynb:2]: <p>have these unit numbers been validated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/573/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Comment (andrewtplatt) [inline examples/use-cases/instruments/FX Forward.ipynb:2]: <p>These numbers refer to the old results, if the new numbers are correct then this explanation needs updating to match the numbers in the table</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/573/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

File: examples/use-cases/instruments/Forward Rate Agreement.ipynb (hunk 2 above)
Comment (peturparkur-fbn) [reply examples/use-cases/instruments/Forward Rate Agreement.ipynb]: Fixed

File: examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb (hunk 1 above)
Comment (peturparkur-fbn) [reply examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1398]: Okay, I have pushed from a clean master (without running it). It seems that is generated from where the file was opened or similar

**Review by SamuelColvinFinbourne — DISMISSED** [review]

**Comment by peturparkur-fbn** [comment]
@rizwansaeed the integration tests need to be re-run. I've fixed a typo

**Review by fbn-seamus — APPROVED** [review]

**Review by rizwansaeed — APPROVED** [review]
